src/utils/env.ts

```diff
@@ -0,0 +1,80 @@
+import { ConfigurationError } from './errors';
+import { logger } from '../logger';
+
+/**
+ * A token, key, or other value which can be stored either in an env file or
+ * directly in the environment
+ */
+export interface RequiredConfigVar {
+  name: string;
+  legacyName?: string;
+}
+/**
+ * Checks the environment for a single variable, taking into account its legacy
+ * name, if app.
+ *
+ * Copies value over to live under current name if only found under legacy name.
+ *
+ * @param envVar a RequiredConfigVar object to check for
+ * @returns true if variable was found, under either current or legacy name,
+ * false otherwise
+ */
+const envHasVar = (envVar: RequiredConfigVar): boolean => {
```

Comment:
Any specific reason why this is not a function?

Reply:
🤔 It is a function... ?

Or do you mean, why didn’t I write it as `function envHasVar(...) { ... }`?

Reply:
Yep, I'm curious why this specific function is not a `function (...)`.
Is it because it's not exported?

Reply:
Nope - nothing other than muscle memory (though I can't promise I'm 100% consistent about that, even so). Happy to switch it back.

Reply:
(I think with the advent of arrow functions I've also been trying to train myself out of using the `function` keyword in callbacks and such, and perhaps that gets over-applied.)

Reply:
Oh, yeah - I see that I was't even consistent within this file. D'oh. Fixing.